frontend/src/upload.js

```diff
@@ -81,28 +103,27 @@ $(document).ready(function() {
   }
 
   // copy link to clipboard
-  $copyBtn.click(() => {
-    if (allowedCopy() && copyToClipboard($('#link').attr('value'))) {
+  $copyBtn.on('click', () => {
+    if (allowedCopy() && copyToClipboard($link.attr('value'))) {
       // record copied event from success screen
       sendEvent('sender', 'copied', {
         cd4: 'success-screen'
       });
 
       //disable button for 3s
       $copyBtn.attr('disabled', true);
-      $('#link').attr('disabled', true);
+      $link.attr('disabled', true);
       $copyBtn.html(
         '<img src="/resources/check-16.svg" class="icon-check"></img>'
       );
       window.setTimeout(() => {
         $copyBtn.attr('disabled', false);
-        $('#link').attr('disabled', false);
+        $link.attr('disabled', false);
```

Comment:
You could potentially move `$link.attr()` up a line and chain both `$copyBtn.attr()` calls.
I think you can also set multiple values in a single `.attr()` by passing an object: http://api.jquery.com/attr/#attr-attributes

```js
$link.attr('disabled', false);
$copyBtn.attr({
  disabled: false,
  'data-l10n-id': 'copyUrlFormButton'
});
```